app/jobs/appointment_confirmed_notification_job.rb

```diff
@@ -0,0 +1,13 @@
+class AppointmentConfirmedNotificationJob < ActiveJob::Base
```

Comment:
Yeah, certainly would be nice to cut down on the boilerplate. I'll do a quick rejig and see how it looks.